Jakefile.js

```diff
@@ -976,6 +976,7 @@ var servicesLintTargets = [
 var lintTargets = compilerSources
     .concat(harnessCoreSources)
     .concat(serverCoreSources)
+    .concat(["client.ts"].map(function(f) { return path.join(serverDirectory, f); }))

```

Comment:
I don't know what all the effects would be. If we fix (many) more lint problems we could just use `harnessSources` (which contains client.ts) instead of `harnessCoreSources` here.